### PR TITLE
update paths to input maps

### PR DESCRIPTION
**Pull request by keskitalo**
Changing the input map paths to the new locations and editing the setup scripts to account for the fact that the combined foreground map no longer has the CMB in it.

### CODE REVIEW

File: dc1/foreground_sim/setup_sim.py [hunk 1]
```diff
@@ -154,9 +134,9 @@
 """
 python -c '
 import numpy as np
-from healpy import read_map;
+from healpy import hp.read_map;
```
Comment (zonca) [inline dc1/foreground_sim/setup_sim.py:137]: import healpy as hp

File: dc1/foreground_sim/setup_sim.py [hunk 2]
```diff
@@ -35,45 +28,32 @@
...
+        fg = hp.read_map(
             fname_in,
             None,
             nest=True,
             verbose=False,
             dtype=np.float32,
         )
         if not os.path.isfile(fname_out):
-            write_healpix(fname_out, m * 1e-6, coord="C", nest=True)
+            write_healpix(fname_out, fg * 1e-6, coord="C", nest=True)
```
Comment (zonca) [inline dc1/foreground_sim/setup_sim.py:44]: can you also add the unit to the output files? In `hp.write_map` is `column_units`

**Comment by zonca** [comment]
You are generating some masks, are they part of the release? in case they are in hdf5, it would better to convert them to FITS

File: dc1/foreground_sim/setup_sim.py (hunk 2 above)
Comment (keskitalo) [reply dc1/foreground_sim/setup_sim.py:44]: Sadly no, the HDF5 format does not recognize that keyword right now. Easy to fix but we don't want to change the revision of TOAST used to run the sims.

**Comment by keskitalo** [comment]
[Good point. I now generate them in both formats.

image

**Review by zonca — APPROVED** [review]

**Pull request merged**